cvat-ui/src/components/annotation-page/standard-workspace/canvas-wrapper.tsx

```diff
@@ -571,7 +571,7 @@ export default class CanvasWrapperComponent extends React.PureComponent<Props> {
                     canvasInstance.fit();
                 }
             }
-            if (activatedState.objectType !== ObjectType.TAG) {
+            if (activatedState && activatedState.objectType !== ObjectType.TAG) {
```

Comment:
How can we get situation when ``activatedStateID !== null``, but it isn't contained in annotations?

Reply:
@bsekachev, have no idea but in some situations we have `annotations` with empty list

Reply:
@ActiveChooN 

Probably we should investigate the root cause. It looks like workaround.
At least need to understand the reason

Reply:
@bsekachev, i guess we can recreate an issue for further investigation and commit this changes to avoid canvas failing. What do you think?

Reply:
@ActiveChooN 

Yep. Nice way. Could you create it?